src/main/java/core/mate/academy/service/MachineServiceImpl.java

```diff
@@ -1,7 +1,42 @@
 package core.mate.academy.service;
 
-/**
- * Your implementation of MachineService.
- */
-public class MachineServiceImpl {
+import core.mate.academy.model.Machine;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+public class MachineServiceImpl implements MachineService<Machine> {
+    private static final String BULLDOZER = "Bulldozer";
+    private static final String TRUCK = "Truck";
+    private static final String EXCAVATOR = "Excavator";
+
+    @Override
+    public List<Machine> getAll(Class<? extends Machine> type) {
+        switch (type.getSimpleName()) {
+            case BULLDOZER: {
+                return new ArrayList<>(new BulldozerProducer().get());
+            }
```

Comment:
{} is optional for casses

Reply:
i thought that without this {} mvn clean package wont pass